src/components/application_manager/src/resumption/resumption_data_json.cc

```diff
@@ -152,32 +160,38 @@ void ResumptionDataJson::IncrementIgnOffCount() {
   LOG4CXX_AUTO_TRACE(logger_);
   sync_primitives::AutoLock autolock(resumption_lock_);
   Json::Value to_save = Json::arrayValue;
-  for (Json::Value::iterator it = GetSavedApplications().begin();
-       it != GetSavedApplications().end();
-       ++it) {
-    if ((*it).isMember(strings::ign_off_count)) {
-      Json::Value& ign_off_count = (*it)[strings::ign_off_count];
+  Json::Value dictionary;
+  resumption::LastStateAccessor accessor = last_state_wrapper_->get_accessor();
+  dictionary = accessor.GetData().dictionary();
+  Json::Value& saved_apps = GetSavedApplications(dictionary);
+  for (auto json_app = saved_apps.begin(); json_app != saved_apps.end();
+       ++json_app) {
```

Comment:
maybe range-based

Reply:
@SNiukalov fixed in https://github.com/smartdevicelink/sdl_core/pull/2626/commits/f4e32226bda388a572a9edc282e74636a66ad925

Reply:
@YarikMamykin i can't see it fixed